examples/yew-tailwindcss/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "yew-example"
+version = "0.1.0"
+authors = ["Anthony Dodd <dodd.anthonyjosiah@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+console_error_panic_hook = "0.1.6"
+wasm-bindgen = "=0.2.74"
+wee_alloc = "0.4.5"
+yew = "0.18"
+
+[features]
+default = []
+demo-abc = []
+demo-xyz = []
```

Comment:
No need to use these demo features, let's just remove them.

Reply:
Done